pom.xml

```diff
@@ -533,6 +533,47 @@ limitations under the License.
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>com.diffplug.spotless</groupId>
+          <artifactId>spotless-maven-plugin</artifactId>
+          <version>2.28.0</version>
+          <configuration>
+            <java>
+              <!-- orders of used formatters are important MPOM-376 -->
+              <!-- eg. palantir override importOrder, so should be first -->
+              <palantirJavaFormat />
+              <removeUnusedImports />
+              <importOrder />
+              <licenseHeader>
+                <content>/*
+ * Copyright The Codehaus Foundation.
```

Comment:
not quite sure all files will have this header and we cannot change few.
https://github.com/codehaus-plexus/plexus-classworlds/blob/master/src/main/java/org/codehaus/classworlds/ClassRealm.java

Reply:
Can we apply multiple header files ? The top one would be the codehaus one (before the `package` declaration) covering any recent change (since the original donation), the original one would be left as is (i.e. after the `package`).

Fwiw, in this very case, I think "the Werken company" was founded by Bob McWhirter in 2002 and dissolved in Dec 2004 (according to his linkedin profile), so we could reach to him to ask if we could remove the copyright (if he's the correct owner).

Reply:
That would not follow the ASF rules: https://www.apache.org/legal/src-headers.html#3party
I'll try to be less restrictive.

Reply:
By default Spotless add new header and preserve old ... as we see we need check for duplicate headers on Maven projects during apply Spotless first time.

Reply:
Good thing here is that this repo not belongs under ASF rules